### PR TITLE
Adds el8 support for custom repo names and repo sources

### PR DESCRIPTION
**Pull request by lorengordon**

### CODE REVIEW

**Comment by lorengordon** [comment]
go codebuild go

File: spel/scripts/amigen8-build.sh [hunk 1]
```diff
@@ -69,8 +68,78 @@ function err_exit {
...
 }
 
+
+# CentOS 8 repos
+DEFAULTREPOS=(
+    baseos
+    extras
+    appstream
+)
+if rpm --quiet -q redhat-release
```
Comment (eemperor) [inline spel/scripts/amigen8-build.sh:78]: Interesting.  This syntax actually works?

**Review by eemperor — COMMENTED** [review]
Looks good.  I just had that one comment about the `if` statement syntax.

File: spel/scripts/amigen8-build.sh (hunk 1 above)
Comment (lorengordon) [reply spel/scripts/amigen8-build.sh:78]: yep, if all you care about is the exit status, and 0 means success and anything else means failure

Comment (lorengordon) [reply spel/scripts/amigen8-build.sh:78]: on redhat, it echos:
```
# if rpm --quiet -q redhat-release; then echo it is redhat; fi
it is redhat
# <new prompt>
```

on centos, no echo:
```
# if rpm --quiet -q redhat-release; then echo it is redhat; fi
# <new prompt>
```

Comment (eemperor) [reply spel/scripts/amigen8-build.sh:78]: 😎

**Review by eemperor — APPROVED** [review]
🚀

**Pull request merged**